Cargo.toml

```diff
@@ -22,4 +22,4 @@ serde_json = "1.0.91"
 
 [dependencies.pyo3]
 version = "0.18.0"
-features = ["chrono", "extension-module"]
+features = ["chrono", "extension-module", "multiple-pymethods"]
```

Comment:
I would really suggest to avoid this feature if it is at all possible because it relies on a somewhat brittle backend depending on "life before main" to collect the impl blocks.

Reply:
In this case, I think it would be possible to just move the macro invocations into the existing impl blocks.

Reply:
Great feedback thx @adamreichold 👍🏼

Reply:
I tried this initially but it isn't too happy, see: https://github.com/PyO3/pyo3/issues/2862

We could avoid decide macros and just write the code manually?

Reply:
Ah yes, you are right, the macros will mess with the `#[pymethods]` proc macro. Sorry for suggesting it.

There are plans to avoid the boiler plate by adding syntax like `#[pyclass(eq, ord)]` to automatically derive the Python methods based on the Rust traits, but it isn't finished yet, c.f. https://github.com/PyO3/pyo3/pull/3195.

In summary, I think open-coding these short methods is the best approach for now. Thanks for making the extra effort.